specs/agents/mobile/events.md

```diff
@@ -0,0 +1,53 @@
+## Mobile Events
+
+This document describes event used by the Mobile SDKs using the [OpenTelementry Event Api](https://github.com/open-telemetry/opentelemetry-specification/blob/0a4c6656d1ac1261cfe426b964fd63b1c302877d/specification/logs/event-api.md).
+Event names will be recording using the `event.name`
+
+
+### `ApplicationLaunch` 
+This event represents the occurrence of app entering the foreground or background or other application lifecycle states.
+
+#### Attributes
+| Name         | Type   | Values                 | Description                                        |
+|--------------|--------|------------------------|----------------------------------------------------|
+| `event.name` | String | `ApplicationLifecycle` | The name of the event                              |
+| `state`      | String | `foreground`           | The lifecycle state the app is transitioning to.   | 
+| `state.last` | String | `background`           | The lifecycle state the app is transitioning from. | 
+
+
+
+### Application Non-Responsive (ANR)
+This event represents the occurrence of an 'ANR' error reported by the Android OS.
+
+#### Attributes
+| Name           | Type | Value | Description |
+|----------------|------|-------|-------------|
+| `event.name`   |      |       |             |
```

Comment:
Can and should we provide the stack trace of an ANR to show where an ANR occurred?

Reply:
We can, and it seems like we should, according to @akhileshpok, so yeah I think we should add it as well.

Reply:
Probably we should send all the attributes defined for exceptions [here](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/semantic_conventions/exceptions.md) so that the UI can show their stacktrace as a regular error.

Reply:
do ANRs result in a crash or are they recoverable errors?